components/skeleton/test/skeleton-group.vdiff.js

```diff
@@ -0,0 +1,235 @@
+import '../demo/skeleton-group-test-wrapper.js';
+import '../demo/skeleton-test-box.js';
+import '../demo/skeleton-test-container.js';
+import '../demo/skeleton-test-heading.js';
+import { expect, fixture, html } from '@brightspace-ui/testing';
+
+const oneSkeletonFixture = html`
+	<d2l-skeleton-group-test-wrapper style="width: 330px">
+		<d2l-test-skeleton-container skeleton></d2l-test-skeleton-container>
+		<d2l-test-skeleton-container></d2l-test-skeleton-container>
+		<d2l-test-skeleton-container></d2l-test-skeleton-container>
+	</d2l-skeleton-group-test-wrapper>
+`;
+
+const noSkeletonFixture = html`
+	<d2l-skeleton-group-test-wrapper style="width: 330px">
+		<d2l-test-skeleton-container></d2l-test-skeleton-container>
+		<d2l-test-skeleton-container></d2l-test-skeleton-container>
+		<d2l-test-skeleton-container></d2l-test-skeleton-container>
+	</d2l-skeleton-group-test-wrapper>
+`;
+
+const nestedGroupsRemoveAllFixture = html`
+	<div style="width: 330px">
+		<d2l-skeleton-group-test-wrapper id="to-remove" skeleton>
+			<d2l-skeleton-group-test-wrapper id="to-remove" skeleton>
+				<d2l-test-skeleton-heading level="1">Heading 1</d2l-test-skeleton-heading>
+				<d2l-test-skeleton-box id="to-remove" skeleton></d2l-test-skeleton-box>
+				<d2l-test-skeleton-container></d2l-test-skeleton-container>
+			</d2l-skeleton-group-test-wrapper>
+			<d2l-skeleton-group-test-wrapper>
+				<d2l-test-skeleton-box></d2l-test-skeleton-box>
+			</d2l-skeleton-group-test-wrapper>
+		</d2l-skeleton-group-test-wrapper>
+	</div>
+`;
+
+describe('d2l-skeleton-group', () => {
+
+	it('all-skeleton', async() => {
+		const elem = await fixture(html`
+			<d2l-skeleton-group-test-wrapper style="width: 330px">
+				<d2l-test-skeleton-container skeleton></d2l-test-skeleton-container>
+				<d2l-test-skeleton-container skeleton></d2l-test-skeleton-container>
+				<d2l-test-skeleton-container skeleton></d2l-test-skeleton-container>
+			</d2l-skeleton-group-test-wrapper>
+		`);
+		await expect(elem).to.be.golden();
+	});
+
+	it('one-skeleton', async() => {
+		const elem = await fixture(oneSkeletonFixture);
+		await expect(elem).to.be.golden();
+	});
+
+	it('no-skeleton', async() => {
+		const elem = await fixture(noSkeletonFixture);
+		await expect(elem).to.be.golden();
+	});
+
+	it('make-one-skeleton', async() => {
+		const elem = await fixture(noSkeletonFixture);
+		const container = elem.querySelector('d2l-test-skeleton-container');
+		container.skeleton = true;
+		await container.updateComplete;
+		await expect(elem).to.be.golden();
+	});
+
+	it('make-one-not-skeleton', async() => {
```

Comment:
@jameswklassen, I think this test wasn't set up correctly before? The previous fixture had no items with skeleton applied, the test wasn't actually doing anything. I changed it here to use the `oneSkeleton` fixture instead.

Reply:
oh good catch! yes this test should have a single element with `skeleton` on it, that gets removed and as a result the whole group un-skeletonizes